apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff

```diff
@@ -0,0 +1,796 @@
+      </trans-unit>
+      <trans-unit id="Reject contact request" xml:space="preserve">
+        <source>Reject contact request</source>
+        <target>Отклонить запрос</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Reply" xml:space="preserve">
+        <source>Reply</source>
+        <target>Ответить</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="SMP servers" xml:space="preserve">
+        <source>SMP servers</source>
+        <target>SMP серверы</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Save" xml:space="preserve">
+        <source>Save</source>
+        <target>Сохранить</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Save (and notify contacts)" xml:space="preserve">
+        <source>Save (and notify contacts)</source>
+        <target>Сохранить (и уведомить контакты)</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Saved SMP servers will be removed" xml:space="preserve">
+        <source>Saved SMP servers will be removed</source>
+        <target>Сохраненные SMP серверы будут удалены</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Scan QR code" xml:space="preserve">
+        <source>Scan QR code</source>
+        <target>Сканировать QR код</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Server connected" xml:space="preserve">
+        <source>Server connected</source>
+        <target>Установлено соединение с сервером</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Settings" xml:space="preserve">
+        <source>Settings</source>
+        <target>Настройки</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Share" xml:space="preserve">
+        <source>Share</source>
+        <target>Поделиться</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Share invitation link" xml:space="preserve">
+        <source>Share invitation link</source>
+        <target>Поделиться ссылкой</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Share link" xml:space="preserve">
+        <source>Share link</source>
+        <target>Поделиться ссылкой</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Show QR code to your contact&#10;to scan from the app" xml:space="preserve">
+        <source>Show QR code to your contact
+to scan from the app</source>
+        <target>Покажите QR код Вашему контакту,
+чтобы сосканировать его в приложении</target>
```

Comment:
you can't say like this in Russian - it means you're the one who'll be doing the scan
```suggestion
        <target>Покажите QR код Вашему контакту для сканирования</target>
```